x-pack/plugins/security_solution/server/lib/siem_migrations/rules/task/agent/graph.ts

```diff
@@ -46,7 +50,7 @@ export function getRuleMigrationAgent({
 
 const matchedPrebuiltRuleConditional = (state: MigrateRuleState) => {
   if (state.elastic_rule?.prebuilt_rule_id) {
-    return END;
+    return 'end';
   }
-  return 'translation';
+  return 'translate';
```

Comment:
Shouldn't it be translationSubGraph?

Reply:
The valid keys are defined in the .`addConditionalEdges` above, if they are the same as the node name, the "condition" is not shown on the graph drawing.
I can move it back if you want.

Reply:
Oh, now I see the conditional edge is mapping the node output name. I tested that locally and I don't see it's adding any value, it would be nice if the return of the conditional function was validated.
But since it's not doing anything, I'd rather keep it without this mapping and use the real node names.